bridge_common/tests/base.py

```diff
@@ -1,6 +1,5 @@
-from oslotest import base
+import unittest
```

Comment:
We are using py.test which doesnt require you to inherit "unittest.TestCase"

please read more http://doc.pytest.org/